toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -136,7 +138,8 @@ intltool-0.51.0-7.cm2.noarch.rpm
 itstool-2.0.6-4.cm2.noarch.rpm
 kbd-2.2.0-1.cm2.x86_64.rpm
 kbd-debuginfo-2.2.0-1.cm2.x86_64.rpm
-kernel-headers-5.15.138.1-4.cm2.noarch.rpm
+kernel-cross-headers-5.15.138.1-5.cm2.noarch.rpm
```

Comment:
Why do we need cross-headers exclusively in toolchain_x86?

Reply:
They are not built for ARM64 because for now we want to keep it simple and start with cross-compiling on the AMD64 architecture.